server/views/pages/barcode/pdf/print-coversheets.njk

```diff
@@ -22,6 +22,9 @@
       <div class="govuk-grid-column-full">
         <h1 class="govuk-heading-xl">Print the coversheets</h1>
 
+<p>Your coversheet will download shortly. Check your downloads folder</p>
```

Comment:
The precise content for this page, and the button label on the "Select Envelope Size" page will be the subject of another ticket.

Reply:
👍 Will this content need to change if JS is not enabled?

Reply:
As long as KH designs some appropriate content we'll be fine! 🙄 
Strictly speaking "your coversheet will download shortly" is a lie if you have JS disabled because it uses JS to click the link on page load. But TBH I think we could get away with it as its so minor.